packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js

```diff
@@ -0,0 +1,21 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    projectId: '<',
+    loadbalancer: '<',
+    region: '<',
+    poolId: '<',
+    goBack: '<',
+    trackAction: '<',
+    trackPage: '<',
+    name: '<',
+    addressIp: '<',
+    port: '<',
+    trackCreateAction: '<',
+    trackCreatePage: '<',
```

Comment:
This is not used

Reply:
trackCreateAction is used but not trackCreatePage.